### PR TITLE
Add event to sync project data more frequently - admin users count + active admin users count

### PR DESCRIPTION
**Pull request by lauriejim**
### What does it do?

I added a tracking event, `didUpdateProjectInformation`.
It has been added in a corn job to send us daily, a piece of full updated information about the project data.
For the first iteration, the number of total admin users, and number of active admin users.

### Why is it needed?

It's needed because this number is currently not accurate on our analytics system.
It's important for us to analyze the correct average amount of admin users per application.
And for Strapi Enterprise Customers, we have to make sure that we track the correct numbers.

### Internal related

Please make sure to review another [internal PR](https://github.com/strapi/tracker/pull/51)

### CODE REVIEW

File: packages/core/admin/server/services/metrics.js [hunk 1]
```diff
@@ -20,8 +20,28 @@ const sendDidChangeInterfaceLanguage = async () => {
...
+  strapi.telemetry.send('didUpdateProjectInformation', {
+    groupProperties: { numberOfActiveAdminUsers, numberOfAdminUsers },
+  });
+}
+
+const startCron = (strapi) => {
+  strapi.cron.add({
+    '0 0 0 * * *' :() => {
+      sendUpdateProjectInformation();
+    }
```
Comment (Marc-Roig) [inline packages/core/admin/server/services/metrics.js:36]: I think the lint is complaining about this bit needing to be a shorthand method
```suggestion
    '0 0 0 * * *': () => sendUpdateProjectInformation()
```

**Review by Marc-Roig — COMMENTED** [review]
happy to see a PR of yours here 🚀 
We need more hands here soo... if you want to code more we have plenty of work 🤓 

Added a minor comment because the lint was failing, everything else looks good to me.

**Review by Marc-Roig — APPROVED** [review]

File: packages/core/admin/server/services/metrics.js [hunk 2]
```diff
@@ -20,8 +20,25 @@ const sendDidChangeInterfaceLanguage = async () => {
...
+  const numberOfAdminUsers = await getService('user').count();
+
+  strapi.telemetry.send('didUpdateProjectInformation', {
+    groupProperties: { numberOfActiveAdminUsers, numberOfAdminUsers },
+  });
+};
+
+const startCron = (strapi) => {
+  strapi.cron.add({
+    '0 0 0 * * *': () => sendUpdateProjectInformation(),
```
Comment (Convly) [inline packages/core/admin/server/services/metrics.js:34]: Could we take the habit of adding a small comment with the cron description?

**Comment by lauriejim** [comment]
Thanks a lot @alexandrebodin and @Marc-Roig 🙏

**Pull request merged**